modules/sdk-coin-apt/src/lib/transaction/customTransaction.ts

```diff
@@ -69,6 +69,7 @@ export class CustomTransaction extends Transaction {
 
```

Comment:
Hey I just noticed utility functions defined inside `convertArgumentByABI`.
Please move them to `modules/sdk-coin-apt/src/lib/utils.ts`